featherbb/Model/Forum.php

```diff
@@ -216,6 +222,7 @@ public function print_topics($forum_id, $sort_by, $start_from)
                     $cur_topic['subject_formatted'] = '<a href="'.Router::pathFor('Topic', ['id' => $cur_topic['id'], 'name' => $url_subject]).'">'.Utils::escape($cur_topic['subject']).'</a> <span class="byuser">'.__('by').' '.Utils::escape($cur_topic['poster']).'</span>';
                     $status_text[] = '<span class="closedtext">'.__('Closed').'</span>';
                     $cur_topic['item_status'] .= ' iclosed';
+                    $cur_topic['icon_type'] = 'icon icon-closed';
```

Comment:
Do you think a moved topic should be flagged as closed, even if it's open in the forum it's been moved in?